packages/analytics/index.tsx

```diff
@@ -42,7 +42,10 @@ export function trackAddToCart(
    const itemcodes =
       addToCartInput.type === 'product'
          ? addToCartInput.product.itemcode
-         : addToCartInput.bundle.items.map((item) => item.itemcode).join(', ');
+         : `${addToCartInput.bundle.currentItemCode}/` +
+           `${addToCartInput.bundle.items
+              .map((item) => item.itemcode)
+              .join(', ')}`;
```

Comment:
The spec has the format `$currentProductSku/$currentProductSku,$relatedSku1,$relatedSku2,...`, and this implementation looks slightly different. Specifically, the current sku might not be the first sku after the `/`, and the delimiter here is `, ` instead of `,`.

Reply:
I think the spec can be a little confusing because there's an explanation for this all the way to the right in the Notes:

> "$currentProductSku in the first position indicates the product page.
> $currentProductSku in the second position should be included in the Event Action only if the product is added to cart.
> 
> SITUATION 1
> Let's assume the event in the report is in the format below:
> Add to Cart - Frequently Bought Together - $currentProductSku/$currentProductSku,$relatedSku1,$relatedSku2
> 
> The meaning:
> Both the current product and the related products were added to cart.
> - currrent product
> - related product 1
> - related product 2
> 
> 
> SITUATION 2
> Now let's assume the event in the report is in this format (without $currentProductSku after the slash):
> Add to Cart - Frequently Bought Together - $currentProductSku/$relatedSku1,$relatedSku2
> 
> The meaning:
> Only the related products of the current product were added to cart. 
> - related product 1
> - related product 2"

I think that should answer the question, but let me know if not!

Reply:
Thank you, I totally missed that! This implementation seems on point then 👍🏻